components/script/dom/element.rs

```diff
@@ -345,16 +345,17 @@ impl Element {
         self.is.borrow().clone()
     }
 
+    /// <https://dom.spec.whatwg.org/#concept-element-defined>
```

Comment:
https://dom.spec.whatwg.org/#concept-element-custom-element-state might be a better link here since `defined` is just one aspect of custom element state, but this method seems to address the general concept.

Reply:
That makes sense, thank you for updating it!